modules/monitoring/alert-rules.nix

```
@@ -202,6 +202,11 @@
     summary = "{{$labels.alias}}: Mail send failed";
     description = "{{$labels.alias}}: Mail send failed";
   };
+  postfix_queue_deferred_messages = {
+    condition = ''postfix_showq_message_size_bytes_count{queue="deferred"} > 1'';
```

```suggestion
    condition = ''postfix_showq_message_size_bytes_count{queue="deferred"} >= 1'';
```
Or do we really only want this to fire when at least 2 messages are deferred? If so, I'd suggest changing the description.

---

Didn't want to alert on the very first mail and I'd say with two messages in the queue the alert description still stands